newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -478,12 +484,7 @@ export default class LayerRenderer {
     this._threeGroup = new THREE.Group();
     threeScene.add(this._threeGroup);
 
-    const threeCamera = new THREE.PerspectiveCamera(
-      this.layer.getCamera3DFieldOfView(),
-      1,
-      0.1,
-      2000
-    );
+    const threeCamera = new THREE.PerspectiveCamera(45, 1, 0.1, 2000);
```

Comment:
why forced to 45?

Reply:
Just a default value that will be overridden at render time :)